### PR TITLE
fix: normalize file path in Algolia index file

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Moved #910 to this PR to force generating a preview.

normalize File Path

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [ ] Changes have been done against dev branch, and PR does not conflict
- [ ] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/939)

### CODE REVIEW

File: js/algolia-index.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ const asciidoctor = Asciidoctor();
 const { convert } = require('html-to-text');
 const git = simpleGit(__dirname);
 
+function resolve(...args) {
```
Comment (VladimirNalivaiko) [inline js/algolia-index.js:10]: I'd suggest to rename this method to `resolvePath()` for a more obvious reading of the code

**Review by lorcan-codes — DISMISSED** [review]

File: js/algolia-index.js (hunk 1 above)
Comment (stoobie) [inline js/algolia-index.js:10]:
```suggestion
function resolvePath(...args) {
```

File: js/algolia-index.js [hunk 2]
```diff
@@ -33,7 +37,7 @@ const startIndexing = (currentBranch) => {
   const algoliaIndex = client.initIndex(
     `${algoliaIndexNamePrefix}-${currentBranch}`
   );
-  const commonPath = path.resolve("../", "components");
+  const commonPath = resolve("../", "components");
```
Comment (stoobie) [inline js/algolia-index.js:40]:
```suggestion
  const commonPath = resolvePath("../", "components");
```

**Comment by stoobie** [comment]
@ylmin @VladimirNalivaiko Thanks for your contributions! I merged this.

**Pull request merged**